readme.org

```diff
@@ -5,6 +5,56 @@ development teams.
 
 [[https://travis-ci.com/jemstep/captain-git-hook][https://travis-ci.com/jemstep/captain-git-hook.svg?branch=master]]
 
+** Build and Install from Source
```

Comment:
A section I'd like to see in this document is a quick start.
Something along the lines of: "Just run these commands and then copy paste this code into this directory and you're good to go!"